sources/api/pluto/src/main.rs

```diff
@@ -160,6 +161,12 @@ mod error {
             instance_type
         ))]
         NoInstanceTypeMaxPods { instance_type: String },
+
+        #[snafu(display("Unable to parse '{}' to a u32", max_num_pods))]
+        StringParseU32 {
+            max_num_pods: String,
+            source: std::num::ParseIntError,
+        },
```

Comment:
This is fundamentally handling the same error as `ParseToU32` above. I would try and generalizing `ParseToU32` to the places it's being used.